refocus-bdk-server.js

```diff
@@ -485,6 +485,22 @@ module.exports = (config) => {
     }, HEARTBEAT_TIMER);
   } // heartBeat
 
+  /**
+   * Checks RoomType bot list for bot triggering event
+   * @param {Object} event - refocusEvent
+   * @param {String} bot - botName
+   * @returns {boolean}
+   */
+  function isBotInstalledInRoom(event, bot) {
```

Comment:
Since you are already looping the botsInRoom, you could by the end of the loop - in the worst case - know if its there or not, then you don't need the include.

I'd suggest find something like that (don't know if it works, didn't run that :P):

```
BOT_NAME = 'botA';
botsInRoom = [{ name:"botA" },{ name:"botA" }];

let obj = botsInRoom.find(o => o.name === BOT_NAME);
if (obj) 
  return true
else
  return false
```

Reply:
or `return botsInRoom.find(o => o.name === BOT_NAME) ? true : false;`

Reply:
cool, I was looking for a way of not needing the extra variable `botsInstalled = []`, maybe that will do, I'll experiment with that.